### DISCUSSION TITLE
nom registry: 404 not found  package 'pnpm' not found

### DISCUSSION THREAD

**Discussion by msipplswezey** [pnpm/pnpm#8845] [Q&A] [answered]
Anyone else getting a 404 when trying to pull pnpm from npm?

Is this an expected change?

https://www.npmjs.com/package/pnpm currently 404s for me since about 10am pacific time this morning.

**Comment by msipplswezey** [comment]
It's back now. 🤷

**Reply by zkochan** [reply, marked as answer]
it was some registry maintenance.

**Reply by msipplswezey** [reply]
💯 🙇
